src/commands/info.rs

```diff
@@ -75,9 +88,15 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
 
     let config = Config::current();
     let (org, project) = config.get_org_and_project_defaults();
+    let org = org.filter(|s| !s.is_empty());
```

Comment:
Dont here instead of in `config.rs` to not alter the current `config-status-json` behavior. If I'd done it there, json would return `null` instead of the current `''`.